### PR TITLE
💬 init room content

### PR DESCRIPTION
**Pull request by k-shar**
https://github.com/WarwickTechCrew/website/issues/5

New pages:
- Avon
- FAB
- Milburn

TODO:
- [ ] Find link to FAB studios info page that tells you what equipment it has
- [ ] find FAB timetable link
- [ ] info about agora
- [ ] Add content about what we do in milburn (is it just a drama rehersal space?)
- [ ] IRL pictures of what they look like 
- [ ] do people like the campus map cause its a bit laggy + big, maybe replace with better images/ directions

### CODE REVIEW

File: wiki/04-spaces/index.md [hunk 1]
```diff
@@ -2,5 +2,27 @@
 description: What spaces we work in and their technical specifications.
 sidebar_custom_props:
     emoji: 🏛️
+resources:
+    - url: https://warwick.ac.uk/services/sg/spa/centraltimetabling/roominformation
+      name: Central Timetabling Website
+      author: Central Timetabling Office
+    - url: https://abs.warwick.ac.uk/wrb2324/default.aspx
```
Comment (adamskrz) [inline wiki/04-spaces/index.md:9]: link to https://warwick.ac.uk/roombooking instead (eternal link)
also add https://warwick.ac.uk/timetablereports for generating more custom timetables

Comment (k-shar) [reply wiki/04-spaces/index.md:9]: Do the custom timetables update? it looks like they just gen a one time snapshot?

Comment (JoshHeng) [reply wiki/04-spaces/index.md]: Wdym? It updates whenever you load the page?

Comment (k-shar) [reply wiki/04-spaces/index.md]: But i mean it only shows the fixed period of time you tell it to doesn't it? Or can you make it track to the current date?

**Pull request merged**